### PR TITLE
Page Responsiveness

### LINKED ISSUE #16: [Rework] Responsiveness of EVERY page
# All Components and Views

#### What is required?
- [x] Wireframe
- [x] HF design
- [x] Working component

#### Additional Notes
Just make sure its all working.

### PR DESCRIPTION
**Pull request by cadrianxyz**
# Description

👍 MERGED WITH EVENT INITIATION 👍 

Include fixes for responsiveness for the following pages:
- Onboarding/welcome (sign in)
- Sign up
- Discover

Would recommend looking through each page and playing around with the most common screen sizes to make sure I didn't miss anything major. 

> Also included a 'utils' file that we can start using from now on for misc functions that are reusable

Fixes #16 

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. If design change, please ensure design change/addition has been approved by a UX Lead or 

- [ ] Unit-Testing (Jest)
- [ ] Visual UX Approval

# Checklist:

- [ ] Code follows the project's style guidelines (no linting set up yet)
- [x] I have performed a self-review of the Code being pushed
- [ ] Code has been reviewed by at least one other developer
- [x] Code has appropriate commenting, particularly in hard-to-understand areas
- [x] Corresponding changes to the documentation have been made
- [x] The changes generate no new warnings

### CODE REVIEW

File: src/components/exploreBar.vue [hunk 1]
```diff
@@ -121,44 +121,48 @@ export default {
 }
 .explore-bar-wrapper {
   position: fixed;
-  font-size: 1em;
+  font-size: 1.5em;
```
Comment (kraytaps) [inline src/components/exploreBar.vue:98]: make nav tabs display flex

Comment (cadrianxyz) [reply src/components/exploreBar.vue:98]: Screen Shot 2020-02-27 at 10 24 34 PM
`display: flex` looks like it causes mobile view to break.. I'm not sure why we might need..?

Comment (kraytaps) [reply src/components/exploreBar.vue:98]: it's nav-links actually but it's already display flex. i wanted to try justify content center but it looks like space evenly as it is is good enough

**Review by kraytaps — APPROVED** [review]
all good

**Pull request merged**